.github/workflows/ci.yml

```diff
@@ -3,7 +3,7 @@ name: CI
 on:
   push:
     branches:
-      - master
```

Comment:
I'm not certain, but I think this will get ignored any you need to add `main` and `version-1` both on the `main` branch. It doesn't hurt to try this though

Reply:
There is at least one ci system that works this way (triggers defined only in main branch), but i think github takes them from the target branch instead. Either way it would have to be in a separate PR

Reply:
I'll merge this and we can see :laughing: